### PR TITLE
Replace ProgressBar with ProgressReport

### PR DESCRIPTION
**Pull request by fabricio-aguiar**
ref #5444
https://pulp.plan.io/issues/5444

Please be sure you have read our documentation on creating PRs:
https://docs.pulpproject.org/en/3.0/nightly/contributing/pull-request-walkthrough.html

### CODE REVIEW

File: CHANGES/5444.misc [hunk 1]
```diff
@@ -0,0 +1 @@
+Replace `ProgressBar` with `ProgressReport`.
```
Comment (bmbouter) [inline CHANGES/5444.misc]: Can this become part of the `5444.removal`? The motivation is that the template we use makes `misc` entries not show the text so the actual changelog won't include this.

**Comment by bmbouter** [comment]
Can you look at the docs section also. If you render the docs locally with `make html` you can see your changes locally. See the entries for ProgresSpinner (for example) here: https://docs.pulpproject.org/en/pulpcore-plugin/nightly/search.html?q=ProgressSpinner&check_keywords=yes&area=default

**Comment by bmbouter** [comment]
@fabricio-aguiar can you also open a PR for the plugin_template. I see at least a few usages there: https://github.com/pulp/plugin_template/search?q=ProgressBar&unscoped_q=ProgressBar

**Comment by fabricio-aguiar** [comment]
> @fabricio-aguiar can you also open a PR for the plugin_template. I see at least a few usages there: https://github.com/pulp/plugin_template/search?q=ProgressBar&unscoped_q=ProgressBar

https://github.com/pulp/plugin_template/pull/105

File: CHANGES/5444.misc (hunk 1 above)
Comment (mdellweg) [reply CHANGES/5444.misc]: In pulp_deb we decided to ignore the show_content flag in the template, because we thought, the misc entries are equally valid.

Comment (bmbouter) [reply CHANGES/5444.misc]: Maybe we should also but another time.

**Pull request merged**